roles/mockturtle/files/sopel_modules/invite.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python
+# vi: set fileencoding=utf-8
+
+import time
+import sopel.bot as bot
+import sopel
+
+@sopel.module.rule(r'.*[Ii]nvite.*')
+@sopel.module.rate(900)
+def invite(bot, trigger):
+    """To new members that are looking for an invitation"""
+    joinedTime = bot.db.get_nick_value(trigger.nick, 'join_channel')
+    diff = time.gmtime() - joinedTime
+    if diff < 3600:
```

Comment:
It's a stylistic (aesthetic) gripe, but would you reverse this check (diff >= 3600) and return early, rather than having the conditional contain the primary action of the function.  It's a preference of mine--you'll find it varies between developers.

Reply:
Do you mean making algebraic op like this, joinedTime - time.gmtime() ?